pom.xml

```diff
@@ -82,7 +82,7 @@
         <neo4j-java-driver.version>4.4.18</neo4j-java-driver.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <!-- to match what is shipped with the Kafka Connect version used in our integration test environment -->
-        <protobuf.version>3.19.6</protobuf.version>
+        <protobuf.version>3.24.4</protobuf.version>
```

Comment:
why do we need the upgrade?

Reply:
version 3.19.6 is not compatible with kafka connect version 7.7.0 but no problems with protobuf 3.24.4 to kafka connect 7.2.9. Another approach would be to change protobuf version according to kafka connect version in integration tests.Wdyt?

Reply:
It's not a big problem as this dependency is used during integration tests only.